delphin/cli/profile_to_dmrs_rdf.py

```diff
@@ -53,25 +53,25 @@ def __cli_parse__(args):
         items = len(ts['item'])
         logger.info(f"Converting {items} items from {args.profile}")
 
-        for row in tsql.select('i-id i-input mrs', ts):
-            id = row[0]
-            text = row[1]
-            encoded = row[2]
-            m = simplemrs.decode(encoded)
-
+        sents_map = {sents['i-id']: sents['i-input'] for sents in ts['item']}
+        mrs_map = {(sent["parse-id"], sent["result-id"]): sent['mrs'] for sent in ts['result']}
+        
+        for ((parses_id, result_id), mrs_string) in mrs_map.items():
+            m = simplemrs.decode(mrs_string)
+            text = sents_map[parses_id]
             # making sure of the well formedness of "m"
             if not is_well_formed(m):
-                logger.warning(f"Item {id} not well formed.")
+                logger.warning(f"Result {result_id} of item {parses_id} not well formed.")
                 # continue
 
             # parse mrs to dmrs and parse it
```

Comment:
I can't understand this comment. Please consider revise or remove.

Reply:
The profile has sentences parsed and analyzed. The `parse-id` camp of the `tsdb` result represents the `id` of the sentence of the profile, the `result-id` is a identifier of the analysis of a particular sentence. This was made to make the reports more meaningful because we can have from one sentence some MRS results which are well formed as  well as some which aren't.
Converting a profile, I found that the same message appeared twice and it was because two analysis of the same sentence were not well formed

Reply:
make it explicit in the comment. Por favor, o mais importante é nos entendermos. Se quiserem, podem usar Portugues.